.github/workflows/build_steps.yml

```diff
@@ -298,6 +303,10 @@ jobs:
           ${{vars.EXTRA_TEST_PREPARE_CMD || ''}}
         env:
           CMAKE_BUILD_PARALLEL_LEVEL: ${{vars.CMAKE_BUILD_PARALLEL_LEVEL}}
+      
+      - name: Install dynamically linked library for tests
+        if: matrix.os == 'linux'
+        run: yum install -y https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/o/openssl11-1.1.1k-7.el7.x86_64.rpm
```

Comment:
Why install it when we're building it with vcpkg?

Reply:
Because the wheel no longer bundle with`openssl` library. Platform which runs `arcticdb` now requires `openssl` manually installed.